client/src/services/team.service.ts

```diff
@@ -80,3 +82,27 @@ export const fetchTeams = async () => {
 
 	return (await res.json()) as WebApi.Entities.Team[];
 };
+
+export const findUsersColleagues = async (id: string, match?: string) => {
+	const result = await callWebApi({
+		method: 'POST',
+		endpoint: 'user/search',
+		body: {
+			id,
+			match
+		}
+	})
+	return await (result.json()) as WebApi.Entities.UserProfile[];
+}
```

Comment:
It looks strange, when you are searching data using POST method.
You can use GET method and query parameters